src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -60,6 +60,13 @@ static int    sched_loop_cnt;
 static struct scheduler g_sched0;
 static struct sched_ops penguin_ops;
 static struct sched_ops ps_ert_ops;
+#define SK_CRASHED -1
+#define SK_ERROR -2
+#define SK_DONE 1
+#define SK_RUNNING 2
+#define AP_DONE 2
+#define AP_START 1
+
```

Comment:
AP_DONE and AP_START is defined in zocl_cu.h

Reply:
done